src/components/stepindicator/StepIndicator/StepIndicator.stories.tsx

```diff
@@ -70,8 +108,13 @@ export const smallCounters = (): React.ReactElement => (
   </StepIndicator>
 )
 
-export const differentHeadingLevel = (): React.ReactElement => (
-  <StepIndicator headingLevel="h2">
+export const differentHeadingLevel = (
+  args: StorybookArguments
+): React.ReactElement => (
+  <StepIndicator
+    headingLevel="h2"
+    ofText={args.ofText}
+    stepText={args.stepText}>
     <StepIndicatorStep label="Personal information" status="complete" />
```

Comment:
Recommend removing the passed in text in one of the tests so that we can ensure the fallback to english works correctly.

Reply:
added